stacks/php-laravel/2.0.1/devfile.yaml

```diff
@@ -28,7 +28,7 @@ components:
         - name: https-laravel
           targetPort: 8000
           protocol: https
-      image: quay.io/devfile/composer:2.5.8
+      image: registry.redhat.io/ubi9/php-83@sha256:7d2c25e76f906aab1e38af8f16ab07ffff9c74e8d3fe9c6b3e8bf9374f9ed057
```

Comment:
I think `registry.access.redhat.com` should be used instead of `registry.redhat.io`

Reply:
btw:
<img width="372" height="99" alt="image" src="https://github.com/user-attachments/assets/347b763d-ba78-45e7-803a-e214ba7a0222" />